### ISSUE TITLE
Event 

### ISSUE THREAD

**Issue by FSAish** [CleverTap/clevertap-react-native#237]
Wanted to check if there is any event that gets fired when we tap on button inApp notification and provide open URL.

` CleverTap.addListener(
      CleverTap.CleverTapInAppNotificationButtonTapped,
      (event: any) => {
        console.log('CleverTapInAppNotificationButtonTapped Event called - ', event);
....`

This log never gets called
I get event only when I add custom key value pair.

**Comment by piyush-kukadiya** [comment]
@FSAish Yes we don't give callback if custom k,v pair is not present. We will discuss this internally if we can support this. Thanks!

**Comment by Mobu34** [comment]
Hello @piyush-kukadiya 

I'm also trying to use this event to get the url when users press any button of our InApp messages but this event never gets called as well and I can't add custom key value pair for InApp campaigns (only for push notifications).
Is it normal ?

Do you plan to support this without custom key value pair ?

Thanks

**Comment by darshanclevertap** [comment]
@Mobu34 You can add KV pairs to the button of the InApp campaigns if you are using the Native InApp templates.

**Comment by piyush-kukadiya** [comment]
@Mobu34 Make sure you are following proper steps given in below docs:

- https://developer.clevertap.com/docs/android-in-app-notifications#in-app-notification-button-onclick-callbacks
- https://docs.clevertap.com/docs/create-message-inapp#compose

You will find it while creating in app campaigns 
<img width="490" alt="Screenshot 2022-09-07 at 1 03 17 PM" src="https://user-images.githubusercontent.com/61137760/188817228-d9d06cf7-41f9-430f-89c2-a512fdd27891.png">
<img width="847" alt="Screenshot 2022-09-07 at 1 03 09 PM" src="https://user-images.githubusercontent.com/61137760/188817236-8dd4c0e1-c385-4853-aa52-731297250bea.png">

**Comment by Mobu34** [comment]
Thanks for replying and for your help @darshanclevertap @piyush-kukadiya 

Actually I could not find this action as I wanted to use the alert template but this action is not available for that one : 
<img width="1346" alt="Capture d’écran 2022-09-07 à 11 10 21" src="https://user-images.githubusercontent.com/68015818/188839862-fed2bcd1-b73d-4a02-905c-421ce8dd7a52.png">

And how suggests FSAish, is it possible to get fired `CleverTap.CleverTapInAppNotificationButtonTapped` without KV pairs ?

Thank you

**Comment by piyush-kukadiya** [comment]
@Mobu34 Without custom kv we don't give callback as of now. Also custom k,v is not applicable for alert in app. Although we can discuss this with our product team if we can have support for custom k,v for alert.
